eta/core/video.py

```diff
@@ -1625,6 +1654,7 @@ def __init__(
             self,
             inpath,
             frames=None,
+            keyframes_only=False,
```

Comment:
I renamed this to `keyframes_only` for consistency with the `frames` argument (its not `in_frames`).

Note also that I was bold and moved this kwarg up in the list. You had it at the end. We are diligent about using `key=value` syntax whenever using keyword arguments, so this is generally safe to do.